### ISSUE TITLE
Is the code based on UnsupervisedMT ?

### ISSUE THREAD

**Issue by Joyxj** [zdou0830/DAFE#2]

**Comment by zdou0830** [comment]
Yes, you are right.

**Comment by Joyxj** [comment]
thanks, I have another question,  the Back-DAFE ,Back-trabslation is on-line ?
thanks again.

**Comment by zdou0830** [comment]
No. We use a pre-trained target-to-source model to perform back-translation offline.

**Comment by Joyxj** [comment]
> No. We use a pre-trained target-to-source model to perform back-translation offline.

Thanks for your answer！

**Comment by Joyxj** [comment]
> Yes, you are right.

Hello, your paper mentions using the out-domian parallel corpus to train the MT system. Is the valid/test set used when you train the MT system out-domain? I'm a little confused.

**Comment by zdou0830** [comment]
We used in-domain valid tests during training.

**Comment by Joyxj** [comment]
> We used in-domain valid tests during training.

Thanks again！
